README.md

```diff
@@ -15,3 +15,14 @@ This repository should be used in conjunction with these following repos :
 ## Starting (locally)
 
 `grunt`
+
+## Seed data
+
+This demo project is seeded with sample user data from [`lib/data.json`](./lib/data.json) upon initialization, make sure to replace this with your own data.
+
+By default all example users' passwords is `'123'`.
+
+# Data reset endpoint
+This service exposes an additional endpoint that is intended for tests and demonstrations at `DELETE /admin/reset`, which causes the underlying data store to be reseeded with the original data for users.
+
+Make sure to deactivate this endpoint when building your own solution on top of this demo, by editing [this file](./lib/routes/admin/index.js).
```

Comment:
Probably better to add an explicit `WARNING: This will delete any data you have created. This is intended for development purposes only. Do not use this for production applications.`